enh(mc): automatic commit - Cisco SSMS REST API (#3114)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Cisco Smart Software Manager On-Prem (formerly known as Cisco Smart Software Manager satellite) is a component of Cisco Smart Licensing
that works in conjunction with Cisco Smart Software Manager (SSM).

> It offers near real-time visibility and reporting of the Cisco licensesyou purchase and consume while giving security-sensitive organizations,

Reviewer: ```suggestion
It offers near real-time visibility and reporting on the Cisco licenses you purchase and consume, while giving security-sensitive organizations
```

> a way to access a subset of Cisco SSM functionality without using a direct Internet connection to manage their install base.

Reviewer: ```suggestion
a way to access a subset of Cisco SSM functionalities without using a direct internet connection to manage their install base.
```

## Pack assets

### Templates

The Monitoring Connector **Cisco SSMS** brings a host template:

* **App-Cisco-Ssms-Restapi-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Cisco-Ssms-Restapi-custom" label="App-Cisco-Ssms-Restapi-custom">

| Service Alias | Service Template                       | Service Description          |
|:--------------|:---------------------------------------|:-----------------------------|
| Alerts        | App-Cisco-Ssms-Alerts-Restapi-custom   | Check minor and major alerts |

> | Licenses      | App-Cisco-Ssms-Licenses-Restapi-custom | Check Licenses               |

Reviewer: ```suggestion
| Licenses      | App-Cisco-Ssms-Licenses-Restapi-custom | Check licenses               |
```

> A service account is required to query the Cisco SSMS API. It must have sufficient read privileges in the account targeted.

Reviewer: 2nd sentence sounds weird, what is it trying to say?

Author: A service account with read-only privileges is required to query the Cisco SSMS API.

> A service account is required to query the Cisco SSMS API. It must have sufficient read privileges in the account targeted.

Author: ```suggestion
A service account with read-only privileges is required to query the Cisco SSMS API.
```
